chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SurveillanceController.java

```diff
@@ -378,15 +471,35 @@ public void streamDocumentContents(@PathVariable("documentId") Long documentId,
         return new ResponseEntity<String>("{\"success\" : true}", responseHeaders, HttpStatus.OK);
     }
 
-    @ApiOperation(value = "Remove documentation from a nonconformity.",
+    @Deprecated
+    @ApiOperation(value = "DEPRECATED.  Remove documentation from a nonconformity.",
             notes = "The logged in user" + " must have either ROLE_ADMIN or ROLE_ACB and administrative "
                     + " authority on the associated ACB.")
     @RequestMapping(value = "/{surveillanceId}/document/{docId}/delete", method = RequestMethod.POST,
             produces = "application/json; charset=utf-8")
-    public String deleteNonconformityDocument(@PathVariable("surveillanceId") Long surveillanceId,
+    public String deleteNonconformityDocumentDeprecated(@PathVariable("surveillanceId") Long surveillanceId,
             @PathVariable("docId") Long docId) throws JsonProcessingException, EntityCreationException,
             EntityRetrievalException, InvalidArgumentsException {
 
+        return deleteNonconformityDocument(surveillanceId, docId);
+    }
+
+    @ApiOperation(value = "Remove documentation from a nonconformity.",
+            notes = "The logged in user" + " must have either ROLE_ADMIN or ROLE_ACB and administrative "
+                    + " authority on the associated ACB.")
+    @RequestMapping(value = "/{surveillanceId}/document/{docId}", method = RequestMethod.DELETE,
```

Comment:
Similarly as with the above "add document to nonconformity". This might be a PUT as it's conceptually an edit of that nonconformity